tensorflow/compiler/xla/service/gpu/tests/hlo_to_llvm_ir.cc

```diff
@@ -75,7 +75,7 @@ xla::Status CompileAndPrintLlvmIr(const std::string& hlo_text,
   std::string target_triple = "nvptx64-nvidia-cuda";
   std::string datalayout = "nvptx64-nvidia-cuda";
   std::string platform_name = "CUDA";
-#else
+#elif TENSORFLOW_USE_ROCM
```

Comment:
in my fixed branch from upstream, this uses #else instread of #elif TENSORFLOW_USE_ROCM, because it actually doesn't call anything from if_rocm_is_configured (BUILD).

And it will cause build fail as well.

Reply:
not sure I follow